frontend/src/libs/utils.ts

```diff
@@ -187,3 +187,14 @@ export function guessActiveSession(sessions: Session[]): Session | null {
 
     return null;
 }
+
+/**
+ * Formats the input "number" string to 2 decimal places.
+ * @param {string} moneyString
+ */
+export function formatMoney(moneyString: string): string {
+    if (!moneyString) {
```

Comment:
What if `moneyString == 0`?

I think it would be better to do an `isFinite` check on the result of `parseFloat`

Reply:
`moneyString` has the type string, so it cannot be 0.